09-scaling-application-workloads/assets/manifests/prometheus-adapter-values-v3.3.1.yaml

```diff
@@ -14,12 +14,12 @@ prometheus:
 replicas: 2
 
 # See: https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/docs/config.md
```

Comment:
Shouldn't this section be commented until the reader gets to the respective section in the guide?

Reply:
Indeed. My bad. I left it uncommented.